platformio.ini

```diff
@@ -20,8 +20,8 @@ extra_configs =
   platformio_esp32_envs.ini
   platformio_special_envs.ini
 
-default_envs = custom_ESP8266_4M1M, custom_ESP32_4M316k
-; default_envs = custom_ESP8266_4M1M
+;default_envs = custom_ESP8266_4M1M, custom_ESP32_4M316k
+default_envs = test_D_ESP8266_4M1M
```

Comment:
The `platformio.ini` file should not be committed unless there are required changes in there. This isn't required.

You'd better expand the desired PlatformIO environment (at least when using VSCode, don't know if Atom also supports that) and execute the Build action from there.

Reply:
Yep Atom has a combobox with the PIO envs in the bottom left.

Reply:
My bad, I thought I removed this from the commit.